__tests__/index.test.js

```diff
@@ -143,6 +144,16 @@ describe('/api/articles', () => {
         expect(response.body.msg).toBe('Topic not found');
       });
   });
+  test('GET:404 sends an appropriate status and error message when queried for a existing topic but with no corresponding articles', () => {
+    const topic = "paper"
+    return request(app)
+      .get('/api/articles')
+      .query({topic})
+      .expect(404)
+      .then((response) => {
+        expect(response.body.msg).toBe('Topic not found');
+      });
+  });
```

Comment:
This should not send an error message but instead an empty array as this is not a bad request